qa/sc_evm_forger_and_delegator_rewards.py

```diff
@@ -0,0 +1,525 @@
+        generate_next_block(sc_node_5, "fifth")
+        generate_next_block(sc_node_6, "sixth")
+        generate_next_block(sc_node_7, "seventh")
+
+        # Reach epoch 79
+        self.advance_to_epoch(79)
+        # Send 100 zen to forger pool mc reward address
+        ft_pool_amount = 1000
+        forward_transfer_to_sidechain(self.sc_nodes_bootstrap_info.sidechain_id,
+                                      mc_node,
+                                      format_eoa(FORGER_POOL_RECIPIENT_ADDRESS),
+                                      ft_pool_amount,
+                                      mc_return_address=mc_node.getnewaddress(),
+                                      generate_block=False)
+        mc_node.generate(self.withdrawalEpochLength - 7)
+        self.sync_all()
+        self.sc_sync_all()
+        sc_last_we_block_id = generate_next_block(sc_node_1, "third node")
+        self.sc_sync_all()
+
+        # Check that the fee distribution took place, exact fees before 1.4 are not tested here
+        api_fee_payments = http_block_getFeePayments(sc_node_1, sc_last_we_block_id)['feePayments']
+        # 7 because the genesis block is counted towards node 1 forger address, not reward address
+        assert_equal(7, len(api_fee_payments))
+
+        # trigger cert submission
+        # Generate 2 SC blocks on SC node and start them automatic cert creation.
+        generate_next_block(sc_node_1, "first node")  # 1 SC block to reach the end of WE
+        generate_next_block(sc_node_1, "first node")  # 1 SC block to trigger Submitter logic
+        # Wait for Certificates appearance
+        time.sleep(10)
+        while mc_node.getmempoolinfo()["size"] < 1 and sc_node_1.submitter_isCertGenerationActive()["result"][
+            "state"]:
+            logging.info("Wait for certificates in the MC mempool...")
+            if sc_node_1.submitter_isCertGenerationActive()["result"]["state"]:
+                logging.info("sc_node generating certificate now.")
+            time.sleep(2)
+        assert_equal(1, mc_node.getmempoolinfo()["size"], "Certificates was not added to MC node mempool.")
+        mc_node.generate(1)
+        self.sync_all()
+        self.sc_sync_all()
+
+        # Generate some blocks for fee distribution calculation
+        generate_next_block(sc_node_1, "first", force_switch_to_next_epoch=True)  # 80
+        ft_pool_amount = 100
+        forward_transfer_to_sidechain(self.sc_nodes_bootstrap_info.sidechain_id,
+                                      mc_node,
+                                      format_eoa(FORGER_POOL_RECIPIENT_ADDRESS),
+                                      ft_pool_amount,
+                                      mc_return_address=mc_node.getnewaddress(),
+                                      generate_block=False)
+        # Activate forger stake v2
+        forger_pool_fee_1, node_1_tip_1 = self.activate_stake_v2(evm_address_sc_node_1, sc_node_1)
```

Comment:
for bettere readability let's add a comment saying this method internally generates 2 more blocks